### DISCUSSION TITLE
Make pylance/pyright supported LS in jupyterlab-lsp

### DISCUSSION THREAD

**Discussion by heejaechang** [microsoft/pylance-release#2919] [Enhancement]
https://github.com/jupyter-lsp/jupyterlab-lsp lets users to configure LSP language server to provider language information.

we should contribute to the repo so that pyright/pylance is one of supported LS. currently, it looks like it supports jedi and builtin python LS.

**Comment by heejaechang** [comment]
tagging @luabud @judej

**Comment by judej** [comment]
Investigate if jupyter-lsp can use Pylance.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
